scripts/apps/monitoring/services/CardsService.ts

```diff
@@ -7,8 +7,19 @@ import {
     SENT_OUTPUT,
     SCHEDULED_OUTPUT,
 } from 'apps/desks/constants';
-import {appConfig} from 'appConfig';
-import {IMonitoringFilter} from 'superdesk-api';
+import {appConfig, extensions} from 'appConfig';
+import {IMonitoringFilter, IPersonalSpaceSection} from 'superdesk-api';
+
+export function getExtensionSections() {
+    const section = flatMap(
```

Comment:
It should be `sections` because it's an array. Actually you don't need this variable at all, you can simply return it's value.